src/index.ts

```diff
@@ -77,27 +108,40 @@ export declare class RaribleSDK {
   public createOrder(order: Order): Promise<Order>;
 
   /**
-   * Gets Sell Orders given a filter.
+   * Update a Sell Order.
+   * The price can only be lowered and not increased,
+   * to increase the price you will need to cancel the order and create a new one.
+   *
+   * @param {Order} sellOrder - sell order.
+   */
+  public updateOrder(sellOrder: Order): Promise<Order>;
+
+  /**
+   * Cancel a Sell Order.
+   * Canceling an order needs to be done on-chain.
+   *
+   * @param {Order} sellOrder - sell order.
+   */
+  public cancelOrder(sellOrder: Order): Promise<void>;
```

Comment:
Hi @benefacto, I added the cancel/update order methods to this PR. Let me know if they look fine to you 👐🏻 
It's pretty simple here, but I believe the implementation won't be so simple haha.

Reply:
@carlosfebres The implementation does not seem like it will be too complicated. Updating Order flow should just be a GET then a PUT to the API. Canceling an Order will just mean passing the Order to the function and waiting for the Cancel event (which is what I think should be returned rather than void).